src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -3,65 +3,65 @@
   "persons" : [ {
     "name" : "Alice Pauline",
     "phone" : "94351253",
-    "email" : "alice@example.com",
+    "email" : "alice@gmail.com",
     "address" : "123, Jurong West Ave 6, #08-111",
     "tagged" : [ "friends" ],
-    "birthday" : "2019-06-05",
-    "income" : "100.00",
-    "riskAppetite": "H"
+    "birthday" : "2018-06-05",
+    "income" : "100000.00",
+    "riskAppetite": "M"
   }, {
     "name" : "Benson Meier",
     "phone" : "98765432",
-    "email" : "johnd@example.com",
+    "email" : "johnd@yahoo.com",
     "address" : "311, Clementi Ave 2, #02-25",
     "tagged" : [ "owesMoney", "friends" ],
-    "birthday" : "2019-05-05",
-    "income" : "100.00",
+    "birthday" : "2017-05-05",
+    "income" : "200.00",
     "riskAppetite": "M"
   }, {
     "name" : "Carl Kurz",
     "phone" : "95352563",
-    "email" : "heinz@example.com",
+    "email" : "heinz@yahoo.com",
     "address" : "wall street",
     "tagged" : [ ],
-    "birthday" : "2019-06-05",
-    "income" : "100.00",
-    "riskAppetite": "H"
+    "birthday" : "2016-06-05",
+    "income" : "25000.00",
+    "riskAppetite": "L"
   }, {
     "name" : "Daniel Meier",
     "phone" : "87652533",
-    "email" : "cornelia@example.com",
+    "email" : "cornelia@gmail.com",
     "address" : "10th street",
     "tagged" : [ "friends" ],
     "birthday" : "2019-06-05",
-    "income" : "100.00",
+    "income" : "70000.00",
     "riskAppetite": "H"
   }, {
     "name" : "Elle Meyer",
     "phone" : "9482224",
-    "email" : "werner@example.com",
+    "email" : "werner@gmail.com",
     "address" : "michegan ave",
     "tagged" : [ ],
-    "birthday" : "2019-06-05",
-    "income" : "100.00",
-    "riskAppetite": "H"
+    "birthday" : "2019-05-05",
+    "income" : "300000.00",
+    "riskAppetite": "L"
   }, {
     "name" : "Fiona Kunz",
     "phone" : "9482427",
     "email" : "lydia@example.com",
     "address" : "little tokyo",
     "tagged" : [ ],
-    "birthday" : "2019-06-05",
-    "income" : "100.00",
+    "birthday" : "2019-06-07",
+    "income" : "150000.00",
     "riskAppetite": "H"
   }, {
     "name" : "George Best",
     "phone" : "9482442",
     "email" : "anna@example.com",
     "address" : "4th street",
     "tagged" : [ ],
-    "birthday" : "2019-06-05",
-    "income" : "100.00",
-    "riskAppetite": "H"
+    "birthday" : "2022-06-05",
+    "income" : "1000000.00",
+    "riskAppetite": "M"
```

Comment:
Nice that you used different data for different contacts! Really helps improve the testing coverage!